mission_control/mission.xml

```diff
@@ -0,0 +1,12 @@
+ <root main_tree_to_execute = "MainTree" >
+
+     <BehaviorTree ID="MainTree">
```

Comment:
@patapufate nit: a comment explaining the mission would be nice

Reply:
Fixed